models/logs.js

```diff
@@ -35,10 +35,22 @@ const fetchLogs = async (query, param) => {
     let call = logsModel.where("type", "==", param);
     Object.keys(query).forEach((key) => {
       // eslint-disable-next-line security/detect-object-injection
-      call = call.where(key, "==", query[key]);
+      if (key !== "limit" && key !== "lastDocId") {
```

Comment:
If condition will be passed for all keys value other than limit and lastDocId,
Why not check which are allowed only?

Reply:
We don't know based on what keys we might want to filter, it can be anything or everything. Instead, I considered excluding the filtering process for `limit` and `lastDocId`